### ISSUE TITLE
After upgrading to DB to later maria-db version, I cannot see my configured reports.

### ISSUE THREAD

**Issue by rfc4711** [Cacti/cacti#5226]
# Describe the bug

After upgrading to DB to later maria-db version, I cannot see my reports. The following error is shown in logs when trying to access the reporting section:

```
2023/02/03 10:35:48 - CMDPHP ERROR: A DB Row Failed!, Error: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'offset, '') AS cint FROM reports LEFT JOIN user_auth ON user_auth.id=reports....' at line 1
2023/02/03 10:35:48 - CMDPHP SQL Backtrace:  (/reports_admin.php[93]:reports(), /lib/html_reports.php[1592]:db_fetch_assoc(), /lib/database.php[449]:db_fetch_assoc_prepared(), /lib/database.php[463]:db_execute_prepared())
2023/02/03 10:39:29 - CMDPHP ERROR: A DB Row Failed!, Error: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'offset, '') AS cint FROM reports LEFT JOIN user_auth ON user_auth.id=reports....' at line 1
2023/02/03 10:39:29 - CMDPHP SQL Backtrace:  (/reports_admin.php[93]:reports(), /lib/html_reports.php[1592]:db_fetch_assoc(), /lib/database.php[449]:db_fetch_assoc_prepared(), /lib/database.php[463]:db_execute_prepared())
```

## To Reproduce

1. Go to Reports

## Expected behavior

Get a list of configured reports.

## Maria-DB:

- OS:  Ubuntu 22.04.1
- Version: mysql  Ver 15.1 Distrib 10.6.11-MariaDB, for debian-linux-gnu (x86_64) using  EditLine wrapper

## Cacti Version:

- OS: Ubuntu 18.04
- Version: Cacti 1.2.1

**Comment by netniV** [comment]
Are you able to pull out the SQL query in question?  It is likely a name that has now become a reserved keyword.

**Comment by TheWitness** [comment]
Very old version.  I suggest that you upgrade as soon as humanly possible.

**Comment by rfc4711** [comment]
@netniV I am not certain if I understand your comment. Do I need to pull the sql query for you? Funny, though, the reports are still being processed and run. Only the web-interface does no longer display them.

**Comment by rfc4711** [comment]
@TheWitness, after upgrading to latest 1.2 version, the error went away.

**Comment by rfc4711** [comment]
Old version caused the SQL query to fail. Upgrade to the latest available resolved the problem.

**Comment by TheWitness** [comment]
Cool.
